packages/grafana-ui/src/components/DataSourceSettings/DataSourceHttpSettings.tsx

```diff
@@ -145,7 +145,13 @@ export const DataSourceHttpSettings: React.FC<HttpSettingsProps> = (props) => {
         <h3 className="page-heading">HTTP</h3>
         <div className="gf-form-group">
           <div className="gf-form">
-            <FormField label="URL" labelWidth={13} tooltip={urlTooltip} inputEl={urlInput} />
+            <FormField
+              className={'gf-form--flex'}
+              label="URL"
+              labelWidth={13}
+              tooltip={urlTooltip}
+              inputEl={urlInput}
+            />
```

Comment:
We should try to move away from the `gf-form` styles and instead use the Form components. I think you can do something like below and if you want to add an constraint to how wide the input should be I would set that to the parent container instead of the input it self.

```suggestion
            <InlineField label="URL" labelWidth={13} tooltip={urlTooltip} grow={true}>
              <Input
                className={inputStyle}
                placeholder={defaultUrl}
                value={dataSourceConfig.url}
                aria-label={selectors.components.DataSource.DataSourceHttpSettings.urlInput}
                onChange={(event) => onSettingsChange({ url: event.currentTarget.value })}
              />
            </InlineField>
```